src/3_PB/SpecificaTecnica/content.typ

```diff
@@ -0,0 +1,653 @@
+- *Urbanistica*: visualizzazione dei dati relativi a parcheggi, colonne di ricarica, zone ecologiche, livello di congestione stradale e batteria delle biciclette elettriche, tramite mappe ed indicatori;
+- *Dati grezzi*: visualizzazione dei dati grezzi provenienti dai sensori, tramite una tabella che mostra i dati in tempo reale, con la possibilità di filtrare per specifico #glossary[sensore] e per tipo;
+- *Superamento soglie*: visualizzazione dei dati relativi al superamento delle soglie di allerta per i diversi tipi di sensori; le allerte vengono visualizzate tramite una tabella che mostra il tipo di #glossary[sensore], il valore rilevato e il timestamp.
+
+=== Sistema di notifica
+#glossary[Grafana] offre un robusto sistema di allerta che consente di rilevare e reagire prontamente a condizioni anomale o critiche nei dati monitorati. Questo sistema non solo permette di definire regole di allerta basate su query ai data source, ma anche di personalizzare le notifiche inviate attraverso vari canali tra i quali Discord.
+Le regole di allerta sono il cuore di questo sistema, consentendo di definire le condizioni che devono attivare un'allerta. Queste regole possono essere configurate per una vasta gamma di scenari; nel nostro contesto di applicazione vengono implementate per rilevare il superamento di determinate soglie, come ad esempio per rilevare temperature eccessive o livelli di inquinantamento atmosferico pericolosi.
+Una volta che un'allerta è attivata, passa attraverso tre stati distinti:
+
+- in attesa (pending): indicando che l'allerta è stata innescata ma la sua conferma è ancora in corso. Questo stato è tipicamente utilizzato per regolare l'invio delle notifiche, assicurandosi che l'allerta sia stabile prima di comunicare il problema;
+
+- attiva (firing): l'allerta è stata confermata e la condizione critica è stata verificata, di conseguenza vengono inviate le notifiche ai canali configurati;
+
+- ok: l'allerta è stata risolta e la situazione è tornata alla normalità.
+
+Per agevolare la gestione delle regole di allerta, #glossary[Grafana]  consente di configurare le regole e i canali di notifica in modo semplice ed efficace tramite la sua interfaccia.
+
+Nel contesto del nostro progetto, abbiamo optato per Discord come canale principale di notifica, configurando il webhook URL e personalizzando i messaggi di notifica per una comunicazione efficace e tempestiva.
+
+Per gestire in modo ancora più sofisticato l'invio delle notifiche, #glossary[Grafana] supporta le "notification policies", consentendo di definire regole specifiche per indirizzare gli alert ai canali di notifica appropriati in base a criteri predefiniti. Questo offre un maggiore controllo e flessibilità nella gestione delle notifiche, garantendo che vengano inviate solo dove e quando necessario.
```

Comment:
non saprei se supparta è il termine giusto in realtà

Reply:
direi piuttosto adopera o mette a disposizione